data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/paths/_eth_v3_validator_blocks_{slot}.json

```diff
@@ -49,6 +42,16 @@
         "example" : "1",
         "format" : "uint64"
       }
+    },{
+      "name" : "skip_randao_verification",
+      "allowEmptyValue" : true,
+      "in" : "query",
+      "schema" : {
+        "type" : "string",
+        "description" : "Skip verification of the `randao_reveal` value (currently ignored by Teku implementation). If this flag is set then the\n `randao_reveal` must be set to the point at infinity (`0xc0..00`). This query parameter\n  is a flag and does not take a value",
```

Comment:
there's a must here - do we care?
i'm wondering if we check that at all, im guessing not. i'd just truncate before 'if this flag is set'
`Skip verification of the randao_reveal value. Ignored in the Teku implementation.`

Reply:
I think we don't care based on this comment: https://github.com/ethereum/beacon-APIs/pull/222#issuecomment-1188455636. Cehcked the other impl blockV2 and blinded_block didn't care about this param.
Changed the description as you suggested.